inc/compatibility/fixes/class-acceleratedmobilepagesfix.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Accelerated Mobile Pages Fix
+ *
+ * @package Pantheon\Compatibility\Fixes
+ */
+
+namespace Pantheon\Compatibility\Fixes;
+
+/**
+ * Accelerated Mobile Pages Fix
+ *
+ * @package Pantheon\Compatibility\Fixes
+ */
+class AcceleratedMobilePagesFix {
+
+
+
+	public static function apply() {
+		SelfUpdatingThemesFix::apply();
+		global $redux_builder_amp;
+		// Force disabling AMP mobile redirection.
+		$redux_builder_amp['amp-mobile-redirection'] = 0;
+
+		if ( ( self::is_mobile() ) && ( false === strrpos( $_SERVER['REQUEST_URI'], 'amp' ) ) ) {
+			header( 'HTTP/1.0 301 Moved Permanently' );
+			header( 'Location: https://' . $_SERVER['HTTP_HOST'] . $_SERVER['REQUEST_URI'] . '/amp' );
+
+			// Name transaction "redirect" in New Relic for improved reporting (optional).
+			if ( extension_loaded( 'newrelic' ) ) {
+				newrelic_name_transaction( 'redirect' );
+			}
+			exit();
+		}
+	}
+
+	/**
+	 * Check if the request is from a mobile device
+	 *
+	 * @phpcs:disable WordPressVIPMinimum.Variables.RestrictedVariables.cache_constraints___SERVER__HTTP_USER_AGENT__
```

Comment:
Was this check triggering warnings for you? There are a lot of VIP checks in [Pantheon WP Coding Standards](https://github.com/pantheon-systems/Pantheon-WP-Coding-Standards) that are just not applicable. I would rather solve it in the source than expressly disable checks inline.